<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win4</class>
 <widget class="QDialog" name="win4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>640</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  
	
	background-color: rgb(232, 39, 47);
  border-width: 10px;
  border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/New folder/bill.png</normaloff>:/new/prefix1/New folder/bill.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/new/prefix1/New folder/MaxPixel.net-Italian-Pizza-Background-Mass-Kitchen-Pizzeria-5163790.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: rgb(216, 31, 45)
}</string>
   </property>
   <property name="text">
    <string>               Choose Your Own</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>45</y>
     <width>110</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/new/prefix1/New folder/Screenshot 2021-07-07 202453.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>640</y>
     <width>191</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  
	
	background-color: rgb(232, 39, 47);
  border-width: 10px;
  border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>Add To Cart</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/New folder/add-to-cart.png</normaloff>:/new/prefix1/New folder/add-to-cart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  
	
	background-color: rgb(232, 39, 47);
  border-width: 10px;
  border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/New folder/cloud-computing.png</normaloff>:/new/prefix1/New folder/cloud-computing.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
  color: white;
  
	
	
	background-color: rgb(34, 62, 102);
  border-width: 3px;
  border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>741</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 39,90%);
color: rgb(176, 176, 176);
border-radius:35px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>150</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>16</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(104, 190, 57);

border-radius:15px;
padding-left:5px;
font: 63 16pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>           Your Cart</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 2px solidrgb(99, 99, 99);
 
  border-radius: 11px;
color: #FFF;
padding-left:12px;
padding-right:5px;
background-color: rgb(39, 39, 39);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 2px solidrgb(99, 99, 99);
 
  border-radius: 11px;
color: #FFF;
padding-left:12px;
padding-right:5px;
background-color: rgb(39, 39, 39);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>560</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 2px solidrgb(99, 99, 99);
 
  border-radius: 11px;
color: #FFF;
padding-left:12px;
padding-right:5px;
background-color: rgb(39, 39, 39);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>590</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 2px solidrgb(99, 99, 99);
 
  border-radius: 11px;
color: #FFF;
padding-left:12px;
padding-right:5px;
background-color: rgb(39, 39, 39);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(104, 190, 57);
border-radius:15px;
padding-left:5px;
font: 25 10pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>Pizaa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(104, 190, 57);
border-radius:15px;
padding-left:5px;
font: 25 10pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(104, 190, 57);
border-radius:15px;
padding-left:5px;
font: 25 10pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>Crust</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(104, 190, 57);
border-radius:15px;
padding-left:5px;
font: 25 10pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>190</y>
     <width>361</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 39,90%);
color: rgb(176, 176, 176);
border-radius:35px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>18</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(104, 190, 57);

border-radius:15px;
padding-left:5px;
font: 63 18pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>                             Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>44</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  
	
	
	background-color: rgb(216, 31, 45);
  border-width: 10px;
  border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/New folder/backmenu.png</normaloff>:/new/prefix1/New folder/backmenu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_4</zorder>
  <zorder>tableView</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>tableView_2</zorder>
  <zorder>label_10</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
